src/Runner.php

```diff
@@ -186,9 +186,15 @@ public function start( $target_dir, $opts ) {
 		Logger::success( sprintf( 'Created %s with composer.lock changes.', $this->getRequestType() ) );
 	}
 
-	private function checkExistingPRBranch() {
+	/**
+	 * Checks to see if there's an existing PR branch.
+	 *
+	 * @param string $type Type of value to return.
+	 * @return mixed
+	 */
+	private function checkExistingPRBranch( $type ) {
 		if ( $this->isGitHub() ) {
-			$cmd = 'hub pr list --format="%t%n" --state=open';
+			$cmd = 'hub pr list --format="%i %t%n" --state=open';
```

Comment:
@danielbachhuber you can specify the output format with a capital I to get the pull request number only (no `#`). You can also use `%H` to get the base branch (e.g. `clu-2019-08-01-00-02`).

I think those would be easier to parse than the entire PR title. You can choose which one to use based on `$type` (`number` or `branch`).

Reply:
See https://hub.github.com/hub-pr.1.html

Reply:
@ataylorme Right now, GitHub output matches that of GitLab output. `lab` doesn't support a `--format` arg, so we can't change the GitHub format.

Reply:
We could break out the `preg_match` logic into a new function that gets the PR number or branch from the specific format and then only call it here when using `lab`. That seems like it might be overkill though if we are going to refactor the entire code base anyway.